functional/tenant-allowlist-sanity/test.sh

```diff
@@ -163,19 +163,19 @@ EOF"
         rlRun "sed 's/^000/111/' allowlist.txt > allowlist2.txt"
         rlRun "gpg --verify allowlist.sig allowlist2.txt" 1
         rlRun -s "lime_keylime_tenant -c addallowlist --allowlist-name list20 --allowlist allowlist2.txt --allowlist-sig allowlist.sig --allowlist-sig-key key.pub" 1
-        rlAssertGrep "Allowlist GPG signature verification failed comparing allowlist (allowlist2.txt) against gpg_sig_file (allowlist.sig)" $rlRun_LOG
+        rlAssertGrep "Allowlist (GPG )?signature verification failed comparing allowlist \(allowlist2\.txt\) against (gpg_)?sig_file \(allowlist\.sig\)" $rlRun_LOG -E
```

Comment:
I would make it even simpler and just search for the phrase "signature verification failed". This snippet is more likely to survive changes than the whole message

Reply:
Fair point. I will update it.